haas/client/user.py

```diff
@@ -0,0 +1,46 @@
+import json
+from haas.client.base import ClientBase
+from haas.client import errors
+
+
+class User(ClientBase):
+    """ Consists of calls to query and manipulate users related
+    objects and relations.
+    """
+
+    def create(self, username, password, privilege):
+        """Create a user <username> with password <password>.
+    <privilege> may by either "admin" or "regular", and determines whether a
+    user is authorized for administrative privileges
+    """
+        url = self.object_url('/auth/basic/user', username)
+
+        if privilege not in('admin', 'regular'):
+            raise TypeError(
```

Comment:
Would suggest ValueError here; TypeError would make more sense if e.g. you passed in an integer.